README.md

```diff
@@ -25,6 +25,14 @@ npm install --save ts-json-schema-generator
 
 Note that different platforms (e.g. Windows) may use different path separators so you may have to adjust the command above.
 
+### Define a CLI command as a npm script for shortcut usage
```

Comment:
I think I'd prefer to show how to use npx.

Reply:
@domoritz
Thank you for the npx idea in the doc. I've updated the readme to use it as an example.